### PR TITLE
Add reference link for Cassandra schema management

### PR DESCRIPTION
**Pull request by utsavoza**
## Which problem is this PR solving?
- https://github.com/jaegertracing/jaeger/issues/4284

## Short description of the changes
- Adds a reference link for Cassandra schema management.

### CODE REVIEW

File: content/docs/1.43/deployment.md [hunk 1]
```diff
@@ -429,6 +429,8 @@ where `{datacenter}` is the name used in the Cassandra configuration / network t
 The script also allows overriding TTL, keyspace name, replication factor, etc.
 Run the script without arguments to see the full list of recognized parameters.
 
+**Note**: See [README](https://github.com/jaegertracing/jaeger/blob/43c29a0e2a331c36e1755e2a6c0f90c1f4181e8e/plugin/storage/cassandra/schema/README.md) for more details on Cassandra schema management.
+
```
Comment (yurishkuro) [inline content/docs/1.43/deployment.md:433]: please make the same change to content/docs/next-release/deployment.md

File: content/docs/1.43/deployment.md [hunk 2]
```diff
@@ -429,6 +429,8 @@ where `{datacenter}` is the name used in the Cassandra configuration / network t
 The script also allows overriding TTL, keyspace name, replication factor, etc.
 Run the script without arguments to see the full list of recognized parameters.
 
+**Note**: See [README](https://github.com/jaegertracing/jaeger/blob/43c29a0e2a331c36e1755e2a6c0f90c1f4181e8e/plugin/storage/cassandra/schema/README.md) for more details on Cassandra schema management.
```
Comment (yurishkuro) [inline content/docs/1.43/deployment.md:432]:
```suggestion
**Note**: See [README](https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/cassandra/schema/README.md) for more details on Cassandra schema management.
```

File: content/docs/1.43/deployment.md (hunk 1 above)
Comment (utsavoza) [reply content/docs/1.43/deployment.md:433]: done

File: content/docs/1.43/deployment.md (hunk 2 above)
Comment (utsavoza) [reply content/docs/1.43/deployment.md]: done

**Review by yurishkuro — APPROVED** [review]
Thanks!

**Pull request merged**